region/lowernorfair/east/Ridley Tank Room.json

```
@@ -66,6 +66,26 @@
       },
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 1],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "h_RModeHeatedCrystalFlashForReserves",
+        {"disableEquipment": "Varia"},
```

In vanilla this would also require removing Gravity. To make this compatible and accurate for all randomizers, you'd need to make a new helper, and its not clear if its worth it's messiness until we are more accurately representing how pauses and disabling works. 
```suggestion
        {"disableEquipment": "Varia"},
        {"disableEquipment": "Gravity"},
```

---

Disabling Gravity, even if unnecessary, is safe to do in this room anyway.